port/unix/omrsysinfo.c

```diff
@@ -6008,7 +6226,16 @@ omrsysinfo_cgroup_is_system_available(struct OMRPortLibrary *portLibrary)
 			}
 			omrthread_monitor_enter(cgroupEntryListMonitor);
 			if (NULL == PPG_cgroupEntryList) {
-				rc = readCgroupFile(portLibrary, getpid(), inContainer, &PPG_cgroupEntryList, &PPG_cgroupSubsystemsAvailable);
+				rc = populateCgroupEntryListV1(portLibrary, getpid(), inContainer, &PPG_cgroupEntryList, &PPG_cgroupSubsystemsAvailable);
+			}
+			omrthread_monitor_exit(cgroupEntryListMonitor);
```

Comment:
If `PPG_cgroupEntryList` is not `NULL`, `rc` is not updated, leaving this function to return `OMRPORT_ERROR_SYSINFO_CGROUP_UNSUPPORTED_PLATFORM`. which is not appropriate.